modules/tinymce/src/core/main/ts/delete/TableDelete.ts

```diff
@@ -154,7 +154,7 @@ const emptyMultiTableCells = (
   return true;
 };
 
-// Runs on a single cell table that has all of its content selected
+// Delete the contents of a range inside a cell. Runs on tables that are a single cell as well as partial selections that need to be cleaned up.
```

Comment:
```suggestion
// Delete the contents of a range inside a cell. Runs on tables that are a single cell and partial selections that need to be cleaned up.
```
The current working just seems to mix the 2 cases together which I found confusing.